src/Failer/DbFailedJobStorage.php

```diff
@@ -156,9 +187,11 @@ public function postProcessor($row)
         $job->connection = $row['connection'];
         $job->queue = $row['queue'];
         $job->error = $row['error'];
+        $job->attempts = (int)$row['attempts'];
         $job->messageClass = $row['messageClass'] ?? null;
         $job->messageContent = $this->connection->fromDatabase($row['messageContent'] ?? null, TypeInterface::ARRAY_OBJECT);
         $job->failedAt = $this->connection->fromDatabase($row['failed_at'], TypeInterface::DATETIME);
+        $job->firstFailedAt = $this->connection->fromDatabase($row['first_failed_at'], TypeInterface::DATETIME);
```

Comment:
Rajouter une valeur par défaut, au cas où la DB n'est pas ma encore migrée : 
`$this->connection->fromDatabase(...) ?? $job->failedAt`

Reply:
Et potentiellement voir si par défaut lors de l'ajout du champ MySQL ne va pas mettre un date time à 0 car non nullable.
Si c'est le cas il faudra aussi vérifier si le timestamps est à 0.